README.md

```diff
@@ -1,35 +1,251 @@
 # Engine Scala Seed
 
 [![Codacy Badge](https://api.codacy.com/project/badge/grade/bc3a79d1b12649158a1eb4758e872141)](https://www.codacy.com/app/Codacy/codacy-engine-scala-seed)
+[![Codacy Badge](https://api.codacy.com/project/badge/Coverage/bc3a79d1b12649158a1eb4758e872141)](https://www.codacy.com/app/Codacy/codacy-engine-scala-seed?utm_source=github.com&utm_medium=referral&utm_content=codacy/codacy-engine-scala-seed&utm_campaign=Badge_Coverage)
 [![Build Status](https://circleci.com/gh/codacy/codacy-engine-scala-seed.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/codacy/codacy-engine-scala-seed)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.codacy/codacy-engine-scala-seed_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.codacy/codacy-engine-scala-seed_2.12)
 
-We use external tools at Codacy, this is the library we use across the multiple external tools integrations.
-For more details and examples of tools that use this project, you can check
-[PMDJava](https://github.com/codacy/codacy-pmdjava),
-[ESLint](https://github.com/codacy/codacy-eslint) and
-[Pylint](https://github.com/codacy/codacy-pylint).
+Framework to help integration with external analysis tools at Codacy.
```

Comment:
Add another sentence explaining the role of these tools, like "These tools provide the issues you can see on Codacy after an analysis is completed".